### PR TITLE
Fixing tr

### PR DESCRIPTION
**Pull request by rosequ**
Fixed the Trec topic reader.

### CODE REVIEW

File: src/main/java/io/anserini/search/query/TrecTopicReader.java [hunk 1]
```diff
@@ -84,7 +84,18 @@ protected StringBuilder read (BufferedReader reader, String prefix, StringBuilde
           k = sb.indexOf(">");
         }
         String title = sb.substring(k+1).trim();
-        // description
+	      
+        //malformed titles, read again
+        if (title.isEmpty()) {
+          sb = read(bRdr,"",null,true,false);	
+          k = sb.indexOf(":");
+                if (k == -1) {
```
Comment (Peilin-Yang) [inline src/main/java/io/anserini/search/query/TrecTopicReader.java]: Could you please make the same indentation with other lines? i.e. 2 spaces

**Comment by Peilin-Yang** [comment]
LGTM

**Comment by Peilin-Yang** [comment]
Oh, I just noticed that you do need a rebase.

**Comment by rosequ** [comment]
Done

**Pull request merged**